gresource: drop to-pixdata preprocessing

Every icon in data/hexchat.gresource.xml is marked
preprocess="to-pixdata", which makes glib-compile-resources shell out to
gdk-pixbuf-pixdata on the BUILD host:

    ../../data/hexchat.gresource.xml: to-pixdata preprocessing requested,
    but GDK_PIXBUF_PIXDATA is not set, and gdk-pixbuf-pixdata is not in PATH.

Our gdk-pixbuf port stages only a cross (i386) gdk-pixbuf-pixdata, which
cannot run on the builder, and the tool was deprecated in gdk-pixbuf 2.42
so it is not dependable on a modern host either.

The icons are still embedded and still compressed="true"; they are stored
as PNG rather than pre-decoded pixdata and GdkPixbuf loads them at
runtime.  Debian and Fedora made the same change when pixdata went away.

--- a/data/hexchat.gresource.xml
+++ b/data/hexchat.gresource.xml
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <gresources>
 	<gresource prefix="/icons">
-		<file alias="hexchat.png" preprocess="to-pixdata" compressed="true">icons/hexchat.png</file>
-		<file alias="book.png" preprocess="to-pixdata" compressed="true">icons/book.png</file>
+		<file alias="hexchat.png" compressed="true">icons/hexchat.png</file>
+		<file alias="book.png" compressed="true">icons/book.png</file>
 
-		<file alias="ulist_voice.png" preprocess="to-pixdata" compressed="true">icons/ulist_voice.png</file>
-		<file alias="ulist_halfop.png" preprocess="to-pixdata" compressed="true">icons/ulist_halfop.png</file>
-		<file alias="ulist_op.png" preprocess="to-pixdata" compressed="true">icons/ulist_op.png</file>
-		<file alias="ulist_owner.png" preprocess="to-pixdata" compressed="true">icons/ulist_owner.png</file>
-		<file alias="ulist_founder.png" preprocess="to-pixdata" compressed="true">icons/ulist_founder.png</file>
-		<file alias="ulist_netop.png" preprocess="to-pixdata" compressed="true">icons/ulist_netop.png</file>
+		<file alias="ulist_voice.png" compressed="true">icons/ulist_voice.png</file>
+		<file alias="ulist_halfop.png" compressed="true">icons/ulist_halfop.png</file>
+		<file alias="ulist_op.png" compressed="true">icons/ulist_op.png</file>
+		<file alias="ulist_owner.png" compressed="true">icons/ulist_owner.png</file>
+		<file alias="ulist_founder.png" compressed="true">icons/ulist_founder.png</file>
+		<file alias="ulist_netop.png" compressed="true">icons/ulist_netop.png</file>
 
-		<file alias="tray_fileoffer.png" preprocess="to-pixdata" compressed="true">icons/tray_fileoffer.png</file>
-		<file alias="tray_highlight.png" preprocess="to-pixdata" compressed="true">icons/tray_highlight.png</file>
-		<file alias="tray_message.png" preprocess="to-pixdata" compressed="true">icons/tray_message.png</file>
+		<file alias="tray_fileoffer.png" compressed="true">icons/tray_fileoffer.png</file>
+		<file alias="tray_highlight.png" compressed="true">icons/tray_highlight.png</file>
+		<file alias="tray_message.png" compressed="true">icons/tray_message.png</file>
 
-		<file alias="tree_channel.png" preprocess="to-pixdata">icons/tree_channel.png</file>
-		<file alias="tree_dialog.png" preprocess="to-pixdata" compressed="true">icons/tree_dialog.png</file>
-		<file alias="tree_server.png" preprocess="to-pixdata" compressed="true">icons/tree_server.png</file>
-		<file alias="tree_util.png" preprocess="to-pixdata" compressed="true">icons/tree_util.png</file>
+		<file alias="tree_channel.png">icons/tree_channel.png</file>
+		<file alias="tree_dialog.png" compressed="true">icons/tree_dialog.png</file>
+		<file alias="tree_server.png" compressed="true">icons/tree_server.png</file>
+		<file alias="tree_util.png" compressed="true">icons/tree_util.png</file>
 	</gresource>
 </gresources>
